assets/js/blocks/checkout/inner-blocks/checkout-billing-address-block/attributes.tsx

```diff
@@ -1,20 +1,13 @@
-/**
- * External dependencies
- */
-import { __ } from '@wordpress/i18n';
-
 /**
  * Internal dependencies
  */
 import formStepAttributes from '../../form-step/attributes';
+import { DEFAULT_TITLE, DEFAULT_DESCRIPTION } from './constants';
 
-export default {
```

Comment:
How come we removed this and now only export a named const?

Should we add `export default attributes` at the bottom of this file?